tests/e2e/ansible/install_test_deps.yaml

```diff
@@ -33,6 +33,48 @@
                 path: bats-core
                 state: absent
           when: bats_exist.rc != 0
+    - block:
+      - name: Check skopeo is installed
+        shell: command -v skopeo >/dev/null 2>&1
+        register: skopeo_exist
+        ignore_errors: yes
+      - name: Install skopeo
+        shell: |
+          sudo apt-get install -y build-essential libgpgme-dev libassuan-dev libbtrfs-dev pkg-config go-md2man
+          git clone https://github.com/containers/skopeo
```

Comment:
I think @fidencio found in https://github.com/confidential-containers/guest-components/pull/669 that we needed a specific version of skopeo in order to get the round trip encryption working? It might be we can get a newer one going, but maybe it's safest to start with that one?

Reply:
well I tried to use the skopeo version `f64a376` but seems like it does not work as when looking for the logs to see that the key is being retrieved by the kbs when doing `kubectl logs -f "$POD"  | grep "$KEY_PATH"`
```
[2024-09-23T17:30:00Z INFO  kbs] Using config file /etc/kbs/kbs-config.toml
[2024-09-23T17:30:00Z WARN  attestation_service::rvps] No RVPS address provided and will launch a built-in rvps
[2024-09-23T17:30:00Z INFO  attestation_service::token::simple] No Token Signer key in config file, create an ephemeral key and without CA pubkey cert
[2024-09-23T17:30:00Z INFO  kbs] Starting HTTP server at [0.0.0.0:8080]
[2024-09-23T17:30:00Z INFO  actix_server::builder] starting 256 workers
[2024-09-23T17:30:00Z INFO  actix_server::server] Tokio runtime found; starting in existing Tokio runtime
```